idunn/utils/categories.py

```diff
@@ -0,0 +1,29 @@
+import os
+from inspect import Parameter
+from apistar import Component
+from .settings import Settings
+from elasticsearch import Elasticsearch
+from .settings import _load_yaml_file
+
+class Categories(dict):
+    """
+        Class to handle the possible categories of places to fetch
+    """
+
+class CategoriesComponent(Component):
```

Comment:
This class is very similar to `IndexNamesSettingsComponent` and `SettingsComponent`.
What about using any abstraction to reduce code duplications ?